skype/skype-ps/skype/Get-CsOnlineUser.md

```diff
@@ -173,6 +173,11 @@ The following updates are applicable for TeamsOnly customers using Microsoft Tea
 - Office
 - Phone
 - WindowsEmailAddress*
+- TeamsEnhancedEncryptionPolicy
```

Comment:
Hello! are these really not relevant to Teams? or do they belong to the filters list? Thanks for clarifying!